### DISCUSSION TITLE
[feature request] optional extra tags for set icon

### DISCUSSION THREAD

**Discussion by kodawah** [andrewgioia/keyrune#256] [General]
Hi,
with all the new promo types using the same set symbol, it would be nice if it was possible to specify a tag to modify the default set icon and append a small letter or symbol. In this way for example one could understand that a card edition is the prerelease one and not the main edition.

I see it used on mkm and a few other sites, i thought it would be nice to have a configuration option to add it.

Example with a default and prerelease (star)
<img width="83" alt="Screen Shot 2020-09-02 at 11 54 45" src="https://user-images.githubusercontent.com/1374357/91967176-dc3c7300-ece0-11ea-846e-a9d094124e37.png">

Example with default, promo pack (pw symbol), prerelease (star), boosterfun (C)
<img width="161" alt="Screen Shot 2020-09-02 at 11 57 11" src="https://user-images.githubusercontent.com/1374357/91967457-4523eb00-ece1-11ea-9681-5174c848ce6d.png">

they don't have to be these symbol necessarily, as long as it's something users can relate to
I hope this feature will be considered a future release, thanks

**Comment by andrewgioia** [comment]
This is a cool idea, I like this a lot. I also like the suggested overlays you put together. 

Could you elaborate a bit on "booster fun"? Are there any others you can think of in addition to that set of 3?

**Comment by kodawah** [comment]
`boosterfun` cards are the "extended art" or "showcase" or "borderless" variants that are found in the collector boosters. While they are technically part of the same edition, most stores have a separate entry for them

I can only think of a "+" variant that sometimes is used for the ultimate master box toppers an the planeswalker decks, as well as "ITA" to indicate the Italian-only legends and the dark editions (but maybe these two should be separate entries entirely)

**Comment by andrewgioia** [comment]
@kodabb got it, I didn't realize "booster fun" was something Wizards uses! Here's the current list with a suggested symbol:

| Name              | Class         | Symbol       | Description                                     |
|-------------------|---------------|--------------|-------------------------------------------------|
| Booster Fun       | `.boosterfun` | smile?       | Borderless, extended art, showcase variants     |
| Promo Pack        | `.promopack`  | planeswalker | Promo pack variant with gold Planeswalker stamp |
| Prerelease Promo  | `.prerelease` | star         | Prerelease promo variant                        |
| Box Toppers       | `.boxtopper`  | box          | Are these unique variants without a set code?   |
| Non-English Print | `.language`   | globe?       | Globe could be too small to identify, possibly [translate](https://thenounproject.com/search/?q=translate&i=7368)            |

There is a set symbol in Keyrune for Italian Renaissance (https://keyrune.andrewgioia.com/icons.html?icon=xren), but I do think having a foreign-language overlay like this could be useful as well.

**Comment by kodawah** [comment]
This is a good list.

A couple of points:
 * I'm not sure about the "smile" symbol, as "boosterfun" is not a widely known term (and having to deal with so many renditions of the same card is not fun at all 😂). I would suggest a plain C for "Collectors" since they come from the collector boosters or a plus like mkm does (https://www.cardmarket.com/en/Magic/Expansions/Throne-of-Eldraine-Extras)
 * non-english print maybe is not appropriate and i gave a bad suggestion: i was only thinking for the dark and legends in italian, which are kinda special-cased. Maybe they'd be better off with a separate set symbol entirely (like xren and xrin)
 * Box Toppers are basically only for UMA right now, but it doesn't hurt to leave it as an option for the future

**Comment by kodawah** [comment]
hi, was there any update to this side project? :)

**Comment by kodawah** [comment]
Another good layer would be having the Etched printing made in a different way.
It should probably be an overlay like the foil effect, since it's just a different finish.

**Comment by andrewgioia** [comment]
Adding to v4

**Comment by andrewgioia** [comment]
Moving this (and all older v4 issues) to discussions!
